packages/schematics/angular/migrations/update-8/update-dependencies.ts

```diff
@@ -49,5 +49,19 @@ export function updateBuilders() {
         },
       );
     }
+
+    const webAnimationsJsVersion = '^2.3.2';
```

Comment:
@mgechev, if the release that they cut is not this version, we need to update this value.